website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**New Features:**
+* You can now enable and manage backup [snapshot distribution](https://www.mongodb.com/blog/post/introducing-snapshot-distribution-atlas) with [`mongodbatlas_cloud_backup_schedule`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule)
+* You can now manage the lifecycle of your [Programmatic API Keys](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Programmatic-API-Keys) [`mongodbatlas_api_key`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/api_key) 
+  *  **IMPORTANT WARNING:**  Managing Atlas Programmatic API Keys (PAKs) with Terraform will expose sensitive organizational secrets in Terraform's state.  We suggest following [Terraform's best practices](https://developer.hashicorp.com/terraform/language/state/sensitive-data).  You may also want to consider managing your PAKs via a more secure method, such as the [HashiCorp Vault MongoDB Atlas Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/mongodbatlas). 
+* You can now retrieve `org_id` from API Keys with [`mongodbatlas_roles_org_id`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/roles_org_id)
+* You can now use the `oplogMinRetentionHours` paramter in [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) and [`mongodbatlas_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster) resources  
```

Comment:
```suggestion
* You can now set the [`oplogMinRetentionHours`](https://www.mongodb.com/docs/upcoming/reference/configuration-options/#mongodb-setting-storage.oplogMinRetentionHours) parameter in [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) and [`mongodbatlas_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster) resources  
```

Reply:
updated, makes sense including link to parameter in our docs. very helpful